Godot Foundation Update 2023 (#761)
godotengine/godot-website: collections/_article/godot-foundation-update-2023.md

---
title: "Godot Foundation Update 2023"
excerpt: "This year has marked significant milestones for the Godot Foundation, and it's all thanks to the incredible support from our community."
categories: ["news"]
author: Godot Foundation
image: /storage/blog/covers/godot-foundation-update-2023.jpg
date: 2023-12-05 12:00:00
---

> This has been a monumental year for the Godot Foundation. We [announced](https://godotengine.org/article/godots-graduation-godot-moves-to-a-new-foundation/) the Foundation one year ago and have been busy since getting all the legalities  in place and transferring over all our assets from the SFC.

Reviewer: ```suggestion
This has been a monumental year for the Godot Foundation. We [announced](https://godotengine.org/article/godots-graduation-godot-moves-to-a-new-foundation/) the Foundation one year ago and have been busy since getting all the legalities in place and transferring over all our assets from the [Software Freedom Conservancy](https://sfconservancy.org/).
```

> On September 12, 2023 we officially [introduced](https://godotengine.org/article/godot-developer-fund/) the [Godot Development Fund](https://fund.godotengine.org/) which we had teased [earlier in the year](https://godotengine.org/article/funding-breakdown-and-hiring-process/). At the time we had an informal goal of reaching €50,000 per month which would allow us to cover our current costs while opening up enough room to hire 1-2 more developers. We reached our goal much faster than expected, we also ended up receiving a sizable one-time donation of [$100,000 from Re-logic](https://twitter.com/Terraria_Logic/status/1704227519027651016).

Reviewer: ```suggestion
On September 12, 2023 we officially [introduced](https://godotengine.org/article/godot-developer-fund/) the [Godot Development Fund](https://fund.godotengine.org/) which we had teased [earlier in the year](https://godotengine.org/article/funding-breakdown-and-hiring-process/). At the time we had an informal goal of reaching €50,000 per month which would allow us to cover our current costs while opening up enough room to hire 1-2 more developers. We reached our goal much faster than expected, and we also ended up receiving a sizable one-time donation of [$100,000 from Re-Logic](https://twitter.com/Terraria_Logic/status/1704227519027651016)! We are incredibly grateful to all the individuals and companies who joined our fundraising effort to make Godot's development sustainable.
```

> On September 12, 2023 we officially [introduced](https://godotengine.org/article/godot-developer-fund/) the [Godot Development Fund](https://fund.godotengine.org/) which we had teased [earlier in the year](https://godotengine.org/article/funding-breakdown-and-hiring-process/). At the time we had an informal goal of reaching €50,000 per month which would allow us to cover our current costs while opening up enough room to hire 1-2 more developers. We reached our goal much faster than expected, we also ended up receiving a sizable one-time donation of [$100,000 from Re-logic](https://twitter.com/Terraria_Logic/status/1704227519027651016).

Reviewer 2: I think you could leave out the "ended up", as it sounds like a, well, ending. Like the fundraising is over!

Author: Yeah, I think I can remove that.

When we launched the funding drive, our largest priorities were and continue to be (in no particular order):

1. More assistance with production tasks including PR review, release management, and team management
2. A dedicated C# maintainer
3. Asset import and asset workflow
4. Web infrastructure, website, asset store
5. Communications, community management, and event planning 

> We have already onboarded André Buse ([winston-yallow](<https://github.com/winston-yallow>)), a long time Godot user and contributor) to focus on web infrastructure. We are currently onboarding Adam Scott ([adamscott](<https://github.com/adamscott>)) to focus on production tasks, and K. S. Ernest Lee ([fire](https://github.com/fire)) to work on asset import and asset workflow. We have people in mind for some of the remaining roles and are currently busy exploring whether that will lead to new hires.

Reviewer 2: ```suggestion
We have already onboarded André Buse ([winston-yallow](<https://github.com/winston-yallow>), a long time Godot user and contributor) to focus on web infrastructure. We are currently onboarding Adam Scott ([adamscott](<https://github.com/adamscott>)) to focus on production tasks, and K. S. Ernest Lee ([fire](https://github.com/fire)) to work on asset import and asset workflow. We have people in mind for some of the remaining roles and are currently busy exploring whether that will lead to new hires.
```
I've had a guess at what you intended with the brackets.

Of course, our current increase in funding is not enough to sustain all these extra roles. At the same time, thanks to the sizable one-time donations we received, we also have savings in the bank that we can use for project-based funding without putting ourselves into a permanent cash flow negative situation. Our idea is to start hiring for short and medium-term contracts in order to address high-priority areas quickly and continue to grow the team with any future increase in monthly donations.

> This year has marked significant milestones for the Godot Foundation, and it's all thanks to the incredible support from our community. To maintain this momentum and achieve more ambitious goals, we would like to invite you to contribute to the [Godot Development Fund](https://fund.godotengine.org/). Your donations will help us to sustain our current growth and enable us to hire more talented individuals, to make the Godot Engine and community the best it can be.

Reviewer: I would add a paragraph for sponsorships, WDYT?
```suggestion
This year has marked significant milestones for the Godot Foundation, and it's all thanks to the incredible support from our community. To maintain this momentum and achieve more ambitious goals, we would like to invite you to contribute to the [Godot Development Fund](https://fund.godotengine.org/). Your donations will help us to sustain our current growth and enable us to hire more talented individuals, to make the Godot Engine and community the best it can be.

If you represent a company interested in joining the Godot Development Fund as a corporate sponsor, [please get in touch](mailto:contact@godot.foundation).
```

Author: Yeah, it is probably a good place. Hopefully we an also add a proper page on the development fund about it as well.
